kapacitor/ast.go

```diff
@@ -281,11 +282,15 @@ type FieldFunc struct {
 	Func  string
 }
 
+var (
+	effRe1 = regexp.MustCompile(`(?Um)\|(\w+)\('(.*)'\)\s*\.as\('value'\)`)
+	effRe2 = regexp.MustCompile(`(?Um)\|eval\(lambda: "(.*)"\)\s*\.as\('value'\)`)
```

Comment:
i'm not sure this is a better change - is this stylistic or does the formmatter need it this way? i think i prefer the way it was because it's easier to have local context - you don't need to scroll up to a variable definition to find the regex that is being used here, it's all right there locally, which at least for me, is easier to follow and reason about.

Reply:
The upside of doing this way is that it compiles the regex once per run vs once per function call, thus saving execution time.